src/lib/deploy.js

```diff
@@ -40,6 +41,30 @@ async function deploy({ alias, yes }) {
     return;
   }
 
+  // Query npm registry to get the latest CLI version.
+  const latestCliVersion = await fetch(
+    'https://registry.npmjs.org/-/package/zkapp-cli/dist-tags'
+  )
+    .then((response) => response.json())
+    .then((response) => response['latest']);
+
+  const globalInstalledPkg = await envinfo.run(
```

Comment:
Reviewing on my phone on a plane. 
Is this envinfo dep the same as we use for 'zk system'?

Reply:
@jasongitmail yes it is the same dependency. No new dependancies were added for this feature.

Reply:
I refactored to not use envinfo.